consensus/hotstuff/voteaggregator/vote_collectors.go

```diff
@@ -106,6 +113,11 @@ func (v *VoteCollectors) getCollector(view uint64) (hotstuff.VoteCollector, bool
 func (v *VoteCollectors) PruneUpToView(lowestRetainedView uint64) {
 	v.lock.Lock()
 	defer v.lock.Unlock()
+	v.log.Info().
+		Uint64("from", v.lowestRetainedView).
+		Uint64("to", lowestRetainedView).
+		Msgf("pruning vote collectors, total: %d", len(v.collectors))
```

Comment:
can we remove this log, and calculate the number of pruned, and add it to the log [`pruned vote collectors, total` here ](https://github.com/onflow/flow-go/pull/2016/files#diff-cf8d024c393d09fb8178e6a6bed7da12e7ff00ab7d8ced31645b47d02ebd7290R148)

Reply:
Of course, but why? I think for logging purposes it's very straightforward and easy to understand. I would like to keep it as simple as possible for this purpose.